components/library/src/content/page.rs

```diff
@@ -282,24 +282,12 @@ impl Page {
     }
 
     /// Creates a vectors of asset URLs.
-    fn serialize_assets(&self, base_path: &PathBuf) -> Vec<String> {
+    fn serialize_assets(&self) -> Vec<String> {
         self.assets
             .iter()
-            .filter_map(|asset| asset.file_name())
+            .filter_map(|asset| asset.strip_prefix(self.file.path.parent().unwrap()).ok())
             .filter_map(|filename| filename.to_str())
-            .map(|filename| {
-                let mut path = self.file.path.clone();
-                // Popping the index.md from the path since file.parent would be one level too high
-                // for our need here
-                path.pop();
-                path.push(filename);
-                path = path
-                    .strip_prefix(&base_path.join("content"))
-                    .expect("Should be able to stripe prefix")
-                    .to_path_buf();
-                path
-            })
-            .map(|path| format!("/{}", path.display()))
+            .map(|filename| format!("{}{}", self.path, filename))
```

Comment:
This is assuming that the logic for pages which used to be much more complex than the logic for sections was unnecessarily complex even before my changes. Both functions are using the same logic after my changes.